components/modal/shared.tsx

```diff
@@ -42,23 +45,58 @@ export const Footer: React.FC<
     onCancel,
     okButtonProps,
     cancelButtonProps,
+    footer,
   } = props;
 
   const [locale] = useLocale('Modal', getConfirmLocale());
 
+  // ================== Locale Text ==================
+  const okTextLocale = okText || locale?.okText;
+  const cancelTextLocale = cancelText || locale?.cancelText;
+
+  // ================= Context Value =================
+  const confirmBtnCtxValue: NormalOkBtnProps = {
+    confirmLoading,
+    okButtonProps,
+    okTextLocale,
+    okType,
+    onOk,
+  };
+
+  const cancelBtnCtxValue: NormalCancelBtnProps = {
+    cancelButtonProps,
+    cancelTextLocale,
+    onCancel,
+  };
+
+  const confirmBtnCtxValueMemo = React.useMemo(
+    () => confirmBtnCtxValue,
+    [...Object.values(confirmBtnCtxValue)],
```

Comment:
这样写行不行呢？（其实我也不太熟悉 23333~）
```suggestion
    Object.values(confirmBtnCtxValue),
```

Reply:
感觉可以，看起来不需要解构

Reply:
你先本地试试， confirmBtnCtxValue如果是 undefined values 是不是会报错

Reply:
这是自己定义的，不会是 undefined